src/jsMain/resources/index.d.ts

```diff
@@ -241,26 +241,26 @@ type MessageReferencedChannels = {
         name: string;
     };
 };
-type MessageDraftOptions = Omit<PublishParameters, "message" | "channel">;
-type SendTextOptionParams = Omit<PublishParameters, "message" | "channel"> & {
+type MessageDraftOptions = Omit<Publish.PublishParameters, "message" | "channel">;
+type SendTextOptionParams = Omit<Publish.PublishParameters, "message" | "channel"> & {
     mentionedUsers?: MessageMentionedUsers;
     referencedChannels?: MessageReferencedChannels;
     textLinks?: TextLink[];
     quotedMessage?: Message;
-    files?: FileList | File[] | SendFileParameters["file"][];
+    files?: FileList | File[] | FileSharing.SendFileParameters<any>["file"][];
 };
-type EnhancedMessageEvent = PubNub.MessageEvent & {
+type EnhancedMessageEvent = Subscription.MessageEvent & {
     userMetadata?: {
         [key: string]: any;
     };
 };
```

Comment:
If this one is something, what is “received” from the listener, then it should be `Subscription.Message`. Also `Subscription.Message` already has: 
```
userMetadata?: {
    [p: string]: Payload;
};
```

```suggestion
type EnhancedMessageEvent = Subscription.Message;
```

Reply:
done